pyproject.toml

```diff
@@ -1,5 +1,57 @@
 [build-system]
-requires = ["setuptools >= 38.0.0",
-            "setuptools_scm >= 2.0.0",
+requires = ["setuptools >= 61.2",
+            "setuptools_scm[toml]>=3.4.3",
```

Comment:
Why did you choose these specific versions as minimums?

Reply:
61.2 is a minimum version of `setuptools` when I felt support of `PEP 621` in it become usable. It got `PEP 621` a few versions earlier, but it used to have a few bugs.

`3.4.3` is the version used to be recommended in `setuptools_scm` docs. Now I have upgraded it to `7`. In `7` `setuptools_scm` got support of making sdists that can be built without git.